compiler/testData/codegen/box/statics/inlineCallsStaticMethodFromOtherPackage.kt

```diff
@@ -0,0 +1,25 @@
+// TARGET_BACKEND: JVM
+// The non-IR backend attempts to call a non-existent accessor in class Test.
```

Comment:
@pyos Could you file an issue for this? With test name note

Reply:
KT-31592